src/store/index.js

```diff
@@ -179,7 +156,65 @@ export default new Vuex.Store({
 
 	},
 	actions: {
+		fetchSummaryToDaysApi({
```

Comment:
Maybe it doesn't seem necessary to create another action here?
`fetchSummaryApi` and `fetchSummaryToDaysApi` calls the same API with the same arguments and shares a lot of repeated codes, only differs partially.

If merging them won't take too much time, I'd suggest you merge these two actions together.

If it would take more than 1~2 hours to do so or there's some good reason against merging it, please comment below.

Reply:
@shc261392 
是可行的，原本也由此想法但由於考量在vuex中封裝action modules 會到致結構變得稍微複雜（輕度黑魔法？而且才一個地方用到）&需要花時間測試，考量到時間因素且該專案沒有很複雜而放棄改以新的action 串接API 。
如果要實作估計2小時左右是足夠的，但會有不確定性存在。

Reply:
經討論後暫緩修改結構為 action modules 

`action 就先跳過吧，其他小地方改完就好
`